examples/datasource-basic/cypress.json

```diff
@@ -1,4 +1,7 @@
 {
-    "video": false,
-    "reporter": "spec"
-}
\ No newline at end of file
+  "video": false,
+  "reporter": "spec",
+  "retries": {
+    "runMode": 2
```

Comment:
**Question:** was this needed because it was flaky otherwise, or it was just added for "safety"?

Reply:
Yes. The test was flaky without apparent reason and most of the time re-running it fixes this flakyness.